pkg/java/jar/parse.go

```diff
@@ -148,7 +148,8 @@ func (p *Parser) parseArtifact(fileName string, size int64, r dio.ReadSeekerAt)
 		case isArtifact(fileInJar.Name):
 			innerLibs, _, err := p.parseInnerJar(fileInJar) //TODO process inner deps
 			if err != nil {
-				return nil, nil, xerrors.Errorf("failed to parse %s: %w", fileInJar.Name, err)
+				log.Logger.Debugf("failed to parse %s: %s", fileInJar.Name, err)
```

Comment:
Debug messages should be capitalized.

```suggestion
				log.Logger.Debugf("Failed to parse %s: %s", fileInJar.Name, err)
```

Reply:
Fixed